lib/manager/npm/extract/index.spec.ts

```diff
@@ -112,6 +112,20 @@ describe('manager/npm/extract', () => {
       );
       expect(res.npmrc).toBeDefined();
     });
+    it('ignores .npmrc when config.npmrc is defined', async () => {
+      fs.readLocalFile = jest.fn((fileName) => {
```

Comment:
We should get rid of those overrides as they tend to cause memory leaks

Reply:
I tried and failed to refactor these. Do you want to try, and if so then now or later?

Reply:
Doing this later